Scripts/fn_toggleLights.sqf

```diff
@@ -0,0 +1,47 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Muta, TyroneMF
+ * Switches Lights on or off in a defined radius.
+ * Call from init.sqf or Trigger.
+ *
+ * Arguments:
+ * 0: Marker <STRING>
+ * 1: Radius <NUMBER>
+ * 2: On <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["LightsOut", 500, true] call FUNC(toggleLights);
+ * ["LightsOut2", 250, false] call tac_Scripts_fnc_toggleLights;
+ */
+
+params ["_marker", "_radius", "_on"];
+
+_targetTypes = [
+    "Lamps_Base_F",
+    "PowerLines_base_F",
+    "Land_LampDecor_F",
+    "Land_LampHalogen_F",
+    "Land_LampHarbour_F",
+    "Land_LampShabby_F",
+    "Land_NavigLight",
+    "Land_runway_edgelight",
+    "Land_PowerPoleWooden_L_F",
+    "Land_FuelStation_01_roof_malevil_F",
+    "Land_fs_roof_F",
+    "Land_Lighthouse_03_base_F",
+    "Land_LightHouse_F",
+    "Land_Airport_Tower_F"
+];
+
+{
+    private _state = if (_on) then {"ON"} else {"OFF"};
+    private _markerPos = getMarkerPos _marker;
+    private _lights = _markerPos nearObjects [_x, _radius];
```

Comment:
`nearObjects` is slow, it will cause a possible stutter.

Reply:
Used it a few times, if it's ran from `init` nobody notices. If it's ran mid-mission with 10-15 players it's not noticable until it's over ~5km (map dependent)

Reply:
You could use `nearEntities` which accepts multiple types in one go. `nearObjects` in this script is the only reason you need 2 loops.

Reply:
`nearEntities` doesn't count map objects, tested in editor by placing a streetlight which was the only one that got switched off.